### PR TITLE
feat: improve advance command

### PR DESCRIPTION
**Pull request by keyvankhademi**

### CODE REVIEW

File: staking/cli/src/instructions.rs [hunk 1]
```diff
@@ -216,15 +226,18 @@ pub fn process_transaction(
             CommitmentConfig::confirmed(),
             RpcSendTransactionConfig {
                 skip_preflight: true,
+                max_retries: Some(10),
```
Comment (guibescos) [inline staking/cli/src/instructions.rs:229]: I know it's counter intuitive but rpc providers usually recommend not setting this

**Review by guibescos — APPROVED** [review]
one comment

File: staking/cli/src/instructions.rs (hunk 1 above)
Comment (guibescos) [reply staking/cli/src/instructions.rs:229]: https://docs.triton.one/chains/solana/sending-txs

Comment (keyvankhademi) [reply staking/cli/src/instructions.rs]: fixed.

**Pull request merged**